app/uk/gov/hmrc/minorentityidentificationfrontend/models/OverseasCompanyDetails.scala

```diff
@@ -79,15 +80,24 @@ object OverseasCompanyDetails {
         }
 
         Json.obj(
-          "VerificationStatus" -> Json.obj("verificationStatus" -> "UNCHALLENGED"),
+          "VerificationStatus" -> checkVerificationStatus(businessVerificationCheck),
           "RegisterApiStatus" -> "not called"
         ) ++ optUtrBlock ++ overseasIdentifiersBlock
       case None =>
         Json.obj(
-          "VerificationStatus" -> Json.obj("verificationStatus" -> "UNCHALLENGED"),
+          "VerificationStatus" -> checkVerificationStatus(businessVerificationCheck),
           "RegisterApiStatus" -> "not called",
-          "isMatch" -> "unmatchable"
+          "isMatch" -> unMatchable
         )
     }
   }
+
+  private def checkVerificationStatus(businessVerificationCheck: Boolean): String = {
```

Comment:
There is a method in AuditHelper that already does something like this so we could use that. It has more statuses but for Overseas Company we never store BV so it would work the same I think

Reply:
the casing in the Strings are slightly different. This matches the confluence page. Potentially update the main auditHelper once we get confirmation of which string to use